llvm/test/Transforms/LoopVectorize/fminimumnum.ll

```diff
@@ -0,0 +1,489 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; FIXME: fmaximumnum/fminimumnum have no vectorizing support yet.
+; RUN: opt --passes=loop-vectorize --mtriple=riscv64 -mattr="+zvfh,+v" -S < %s | FileCheck %s --check-prefix=RV64
+; RUN: opt --passes=loop-vectorize --mtriple=aarch64 -mattr="+neon" -S < %s | FileCheck %s --check-prefix=ARM64
+; RUN: opt --passes=loop-vectorize --mtriple=x86_64 -S < %s | FileCheck %s --check-prefix=X64
```

Comment:
Cannot share targets in a test like this, this is going to break if this full set of targets is not built. This needs to be duplicated in each target subdirectory

Reply:
Looks like this comment wasn't addressed even though the comment was marked as resolved. There should be target independent tests, passing `-force-vector-width=` and the target specific tests need to be in the target-specific subdirectories.